tests/azure/storageaccount_test.go

```diff
@@ -0,0 +1,57 @@
+package azure_test
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"testing"
+
+	"github.com/NeowayLabs/klb/tests/lib/azure"
+	"github.com/NeowayLabs/klb/tests/lib/azure/fixture"
+	"github.com/NeowayLabs/klb/tests/lib/nash"
+)
+
+func genStorageAccountName() string {
+	return fmt.Sprintf("klbstortests%d", rand.Intn(1000))
+}
+
+func testStorageAccountCreate(t *testing.T, f fixture.F) {
+	genstorage := genStorageAccountName()
+	nash.Run(
+		f.Ctx,
+		t,
+		"./testdata/create_storage_account.sh",
+		f.ResGroupName,
+		genstorage,
+		f.Location,
+	)
+	storage := os.Getenv("STORAGE_ACCOUNT_NAME")
```

Comment:
@kamilash one problem with using os.Getenv is that if some other script manipulates it too we are going to have problems since the tests are parallel. It would be good to use the nash API to get a local variable value instead of using setenv, if possible.

@tiago4orion really looking forward to better alternatives :-)